modules/sitemaps/sitemap-constants.php

```diff
@@ -7,6 +7,16 @@
  * @author Automattic
  */
 
+/**
+ * Number of seconds between sitemap and news updates in development code.
```

Comment:
I think it may be worth expanding on that a bit, so folks can quickly understand that this is meant to change default settings that are normally set downer in the file.

Reply:
I've updated the comment. Should you wish to alter it further, please do so (rather than send back to me).